wordpress-mobile/WordPress-iOS
# Change Password: iOS 12 password generator

Fixes part of #10031 

This PR enables the password generator when a user wants to change the password.

![img_2260](https://user-images.githubusercontent.com/912252/51041041-e8763880-15b0-11e9-83bb-11bb088829eb.PNG)

**To test:**
• Select the tab Me -> Account Settings
• Select Change Password
• You should see the iOS 12 password generator
